### PR TITLE
update version 8 LTS start year

### PR DESCRIPTION
**Pull request by cjihrig**

### CODE REVIEW

File: schedule.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
   },
   "v8": {
     "start": "2017-04-30",
-    "lts": "2016-10-31",
+    "lts": "2017-10-31",
```
Comment (richardlau) [inline schedule.json]: The readme currently says `2017-10-01`. These should be consistent.

Comment (hbetts) [inline schedule.json]: `schedule.ical` also refers to `2017-10-01`

Comment (gibfahn) [inline schedule.json]: @nodejs/lts should this be `10-01` or `10-31`? Either way let's fix and land.

Comment (MylesBorins) [inline schedule.json]: I think we should specify the month and no date. update day closer to

**Comment by jasnell** [comment]
+1 to omitting the day. Just list the month and year

File: schedule.json (hunk 1 above)
Comment (hbetts) [inline schedule.json]: +1 for emitting the day. Feels too specific, especially since LTS does not guarantee the LTS version will be cut on that day.

**Comment by gibfahn** [comment]
@nodejs/lts any objections to just landing this as is? The question about day of the month is likely to be a moot point once/if https://github.com/nodejs/LTS/pull/141 lands.

**Pull request merged**